### DISCUSSION TITLE
When to merge

### DISCUSSION THREAD

**Discussion by stratofax** [TrustTheVote-Project/BallotLab#90] [Ideas]
Since I incorporated the ballot generation code from an earlier version of BallotLab into the main program on my development branch (`dev-make-flat`) on my fork of the BallotLab repo, all the GitHub Actions (automated tests) are failing again -- even though all of the same pytests pass on my local development workstation. 

This has to do with accessing some of the PNG files that I need to create the Instructions section on the ballot. For details see my bug report issue, [GitHub Actions fail when attempting to locate external files · Issue #78 · TrustTheVote-Project/BallotLab](https://github.com/TrustTheVote-Project/BallotLab/issues/78)

I'm going to at least make an attempt to fix this issue before I think about merging my branch from my fork back onto the TrustTheVote repo. Unfortunately, this means I can't close any issues at all, since the rule is that you can't close an issue until you merge your code onto `main`. 

But unless this error is a huge problem that's difficult to resolve, I'd like the GitHub Actions to run green across the board before I attempt to merge into the TrustTheVote repo.

**Comment by stratofax** [comment]
@ion-oset the results of this discussion will affect when you and I can merge our code back onto the TTV repo. I think your use of a gist for now means I can continue to work on my fork, until I resolve this issue, but let me know if you see any problems with this approach over the next week or so.

**Reply by trustthevote** [reply]
I'm more focused on having code complete in *any* branch that makes the 4 ballots we need from the 1 EDF that we have. I appreciate the need to get code merges right too of course, but external testers have already started work, and we do not yet have outlook on when we can tell them the real ballots will be available.

**Reply by stratofax** [reply]
I completely agree, @trustthevote -- that's why I haven't even attempted a merge from my fork, where I'm doing all my work. Right now, @ion-oset is sending me code via a gist, which means we don't have to worry about merges to share code, either. I wanted @ion-oset to know that this "gist for the mill" approach is working for me right now, and I'll put off any merges until we reach a point where the code is stable and we can close a bunch of issues.
